src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/equalshashcode/InputEqualsHashCode.java

```diff
@@ -6,40 +6,40 @@
 
 package com.puppycrawl.tools.checkstyle.checks.coding.equalshashcode;
 
-public class InputEqualsHashCode { // ok
```

Comment:
The file lgtm....
Please go through the Issue #13213  description before sending PR.
You will have to remove the respective suppression from  [checkstyle-input-suppressions.xml](https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle-input-suppressions.xml)